pom.xml

```diff
@@ -83,6 +83,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.amazon.redshift</groupId>
```

Comment:
we can put it libs-ext-release at repository.liveramp.com if it's not in maven central (we use it here).  checking in jars with the project is pretty awful.